Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+edition = "2021"
+name = "big-int"
+version = "0.1.0"
+
+[dependencies]
+eyre = "0.6"
```

Comment:
`eyre` is not used in the library anymore, you can move it to `dev-dependencies` if you still want to use it there.

Reply:
ok